modules/services/localization/src/main/java/au/com/shiftyjelly/pocketcasts/localization/helper/RelativeDateFormatter.kt

```diff
@@ -22,18 +22,14 @@ import java.util.Locale
 class RelativeDateFormatter(val context: Context) {
 
     private val sevenDaysAgo: Calendar = Calendar.getInstance().apply { add(Calendar.DATE, -7) }
+    private val sevenDaysFromNow: Calendar = Calendar.getInstance().apply { add(Calendar.DATE, 7) }
```

Comment:
Can we also update the comment above to say?

`* "Monday" if less than a week from now`